### PR TITLE
PLAN-267 - Implementing a first version of the forsys results

### PR DESCRIPTION
**Pull request by george-silva**
Implements running forsys by fetching data from our database.

## What is done:

* given an scenario id and it's configuration
* reading all the data from the database (stands, stand metrics, scenario configuration, planning area, etc)
* forsys runs with:
   * single priority questions
   * multi priority questions
* storing the results in the backend, using `upsert`
* csv's being stored in forsys default directory
* cost per acre is being calculated based on a fixed value, provided by Michele

## What needs to be done

* use `tryCatch` in forsys script to prevent errors from not updating the data
* collect run details and store them along the results as well
* exclusion of stands based on the UI parameters inside the scenario config - we don't have the data in the database yet
* stand thresholds and global thresholds (these include distance from roads and slope). we don't have the data in the database yet
* pcp/spm normalization - from the Forsys docs, looks like it's not required, but I'm digging to see if it's necessary. The docs mention it's already executed for our priorities
* any other normalization of data.
* Move the CSVs stored by Forsys to another safe location, so we can later serve them from an API

### CODE REVIEW

**Review by sachiyuan — APPROVED** [review]
LGTM

File: src/planscape/rscripts/forsys.R [hunk 1]
```diff
@@ -10,9 +10,12 @@ library("glue")
 library("forsys")
 library("sf")
 library("dplyr")
+library("purrr")
```
Comment (runforyourlives) [inline src/planscape/rscripts/forsys.R:13]: Do we need to update the getting started wiki, which contains a list of R libs to install?

Comment (runforyourlives) [reply src/planscape/rscripts/forsys.R:13]: bEvcrnYmh4fhRsX

File: src/planscape/rscripts/forsys.R [hunk 2]
```diff
@@ -10,9 +10,12 @@ library("glue")
...
 
 readRenviron("src/planscape/planscape/.env")
 
+COST_PER_ACRE <- 2470
```
Comment (runforyourlives) [inline src/planscape/rscripts/forsys.R:17]: We're still eventually allowing users to configure this, correct?  Can you leave a TODO to change this later?

Comment (runforyourlives) [reply src/planscape/rscripts/forsys.R:17]: sg

File: src/planscape/rscripts/forsys.R (hunk 1 above)
Comment (runforyourlives) [reply src/planscape/rscripts/forsys.R:13]: Yes, another PR is fine.

File: src/planscape/rscripts/forsys.R [hunk 3]
```diff
@@ -167,97 +267,109 @@ get_configuration <- function(scenario) {
...
       weights = configuration$weights,
       new_field = "priority"
     )
     scenario_priorities <- c("priority")
   } else {
-    scenario_priorities <- names(conditions)
+    scenario_priorities <- first(priorities$condition_name)
   }
-
+  number_of_projects <- 10
```
Comment (runforyourlives) [inline src/planscape/rscripts/forsys.R:289]: can you leave a comment at the top of this assumption and other things we might allow users to specify in the future?  (this is a nonblocking request.)

**Review by runforyourlives — APPROVED** [review]

**Pull request merged**